to_do_list/src/Components/SimpleDialog.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "../Style/SimpleDialog.css";
+
+function SimpleDialog(props) {
+  const { onClose, open, title, action } = props;
+
+  const handleClose = shouldUpdate => {
+    onClose(shouldUpdate, selectedValue);
+  };
+
+  let selectedValue = "";
+  return (
+    <dialog className="dialog" open={open} onClose={handleClose}>
```

Comment:
`<dialog>` tag isn't supported by safari browser, what will i do if i use my macbook in your app?
like the h1's, just make a `<div>` or `<span>` and add your css.